tests/test_logger.py

```diff
@@ -0,0 +1,94 @@
+"""Unit tests for logger module."""
+
+import unittest
+from unittest import mock
+import logging
+
+from PyQt5.QtCore import QObject
+from PyQt5.QtWidgets import QApplication
+
+from iquip.apps import logger
+
+class TestloggerApp(unittest.TestCase):
+    """Unit tests for loggerApp class."""
+
+    def setUp(self):
+        self.qapp = QApplication([])
+
+    def tearDown(self):
+        del self.qapp
+
+    def test_handler_connection(self):
```

Comment:
This test is a unit test for `LoggingHandler`, not `LoggerApp`.
Please move this into another test class.

Reply:
Please remove this test!